### PR TITLE
Expose samples_per_plugin cli arg in README

### PR DESCRIPTION
**Pull request by TimZaman**
* Motivation for features / changes

* Technical description of changes

* Screenshots of UI changes

* Detailed steps to verify changes work correctly (as executed by you)

* Alternate designs / implementations considered

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -339,7 +339,7 @@ custom plot will appear in the TensorBoard image tab.
 TensorBoard uses [reservoir
 sampling](https://en.wikipedia.org/wiki/Reservoir_sampling) to downsample your
 data so that it can be loaded into RAM. You can modify the number of elements it
-will keep per tag in
+will keep per tag by using the `--samples_per_plugin` command line argument (ex: `--samples_per_plugin images=100`). Alternatively you can change the source code in
```
Comment (wchargin) [inline README.md]: Please reflow this line to wrap at 80 characters, just like the
surrounding text.

Comment (wchargin) [inline README.md]: Please include an example that indicates that this is a comma-separated
list: e.g., `--samples_per_plugin=scalars=500,images=20`.

Comment (wchargin) [inline README.md]: nit: comma after “Alternatively”

**Comment by TimZaman** [comment]
Addressed comments!

**Pull request merged**